doc/classes/VisualServer.xml

```diff
@@ -80,6 +83,7 @@
 			<argument index="4" name="z_far" type="float">
 			</argument>
 			<description>
+				Set camera to use frustom projection. This mode allows adjusting the "offset" argument to create "tilted frustum" effects.
```

Comment:
Typo, frust**u**m.
Also for arguments, we usually use `[code]` markup when referring to their identifier.

Reply:
Should also be "Sets" instead of "Set" (same for other methods below).